scripts/release.sh

```diff
@@ -53,13 +53,24 @@ fi
 
 # Build Docker server image
 if [[ -z "$SKIP_DOCKER" ]]; then
-  docker build -t $docker_name:$version .
-  docker build -t $docker_name:$version-standalone --build-arg STANDALONE=1 .
+  docker build -t $docker_name:$version -f amd64.Dockerfile .
```

Comment:
Could you name this `$docker_name:$version-amd64`, then later alias `$docker_name:$version` to it using `docker tag`?

Reply:
Maybe keep the amd64 dockerfile named simply as `Dockerfile`, so that it gets picked up as the default for `docker build` etc? Or perhaps symlink Dockerfile -> amd64.Dockerfile?